akka-stream/src/main/scala/akka/stream/IOResult.scala

```diff
@@ -79,4 +79,9 @@ final case class AbruptIOTerminationException(ioResult: IOResult, cause: Throwab
  * @param count The number of bytes read/written up until the error
  * @param cause cause
  */
-final class IOOperationIncompleteException(val count: Long, cause: Throwable) extends RuntimeException(cause)
+final class IOOperationIncompleteException(message: String, val count: Long, cause: Throwable)
+    extends RuntimeException(message, cause) {
+
+  def this(count: Long, cause: Throwable) = this(null, count, cause)
```

Comment:
really use `null` as the exception message? can't we use something general?

Reply:
Oh, this constructor just to keep binary compatibility, and that's what you got with the previous `RuntimeException(cause)`, so, part of the `Exception` API having `null` to signify no message.

Reply:
But no message was bad before as well :) So, maybe let's just give it some default message like `"IO operation was stopped unexpectedly after ${count} bytes because of ${cause.getMessage}"`

Reply:
Fair enough